src/js/chartAuthoring.js

```diff
@@ -237,6 +360,24 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             $(this).find(dataEntryLabelSelector).val(currentData.label).trigger("change");
             $(this).find(dataEntryValueSelector).val(currentData.value).trigger("change");
         });
+
+        that.events.onUpdateDataEntryPanel.fire();
```

Comment:
Right, your concern is reasonable. In that case, shall we use a event name that describes what had happened when this event is fired, such as "onDataApplied" or "onDataAppliedToEntryPanel".